lblod/detail_fetcher.py

```diff
@@ -5,16 +5,15 @@
 import uuid
 from helpers import logger
 from lblod.job import update_task_status
-from constants import TASK_STATUSES
-from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry # type: ignore
+from constants import TASK_STATUSES, FEATURE_SKIP_UNEXPECTED_RESPONSE_FROM_SOURCE
 import time
 api_url = os.environ["API_URL"]
 
 
-def fetch_detail_url(access_token, v_code, task):
+def fetch_detail_url(v_code, task):
```

Comment:
I made an attempt here at simplifying a bit the flow of control.
https://github.com/lblod/harvesting-verenigingen-scraper-service/pull/25